docs/plugins.md

```diff
@@ -261,7 +260,7 @@ This means if two transforms both visit the "Program" node, the transforms will
 - Plugin ordering is first to last.
 - Preset ordering is reversed (last to first).
 
-```js
+```json
 "plugins": [
   "transform-decorators-legacy", // will run first
```

Comment:
@raspo posted earlier that they still render fine: https://github.com/babel/babel.github.io/pull/1066#discussion_r91809624, but it might be better to re-word some of these areas. In this particular case maybe something like:

====

This means if two transforms both visit the "Program" node, the transforms will run in either plugin or preset order.

* Plugins run before Presets.
* Plugin ordering is first to last.
* Preset ordering is reversed (last to first).

For example:

```json
{
  "plugins": [
    "transform-decorators-legacy",
    "transform-class-properties"
  ]
}
```

Will run `transform-decorators-legacy` then `transform-class-properties`.

It is important to remember that with presets, the order is _reversed_. The following:

```json
{
  "presets": [
    "es2015",
    "react",
    "stage-2"
  ]
}
```

Will run in the following order: `stage-2`, `react`, `es2015`.

This is mostly the result of ensuring backwards compatibility, since most users list "es2015" before "stage-0". For more information, see [notes on potential traversal API changes](https://github.com/babel/notes/blob/master/2016-08/august-01.md#potential-api-changes-for-traversal).

====

Thoughts?